docs/getting-started/installation.md

```diff
@@ -18,6 +18,24 @@ Downloads `.tar.gz`:
 
 Checkout all builds on [github](https://github.com/quickwit-inc/quickwit/releases)
 
+### Note on external dependencies
```

Comment:
Nice section. It would be nice to put the link of the static binary packages above.